### ISSUE TITLE
Not enough values error in creating the aggregated epitopes report

### ISSUE THREAD

**Issue by nruthen** [griffithlab/pVACtools#660]
* pvactools version: 2.0.0 (Using docker image griffithlab/pvactools:2.0.0)
* Python version: 3.7.0 (From python base image)
* Operating System: Debian GNU/Linux 10 (buster)

**Describe the bug**
I ran into an error similar to Issue #553 while running the pVACseq pipeline on VEP-annotated vcf files, however, this error occurred while the aggregated epitopes output was being generated.

**To Reproduce**
pVACtools command (via a bash script run within the docker container):

```
pvacseq run \
	-e1 8,9,10,11 \
	-e2 12,13,14,15,16,17,18 \
	--binding-threshold 1000 \
	--iedb-install-directory "/opt/iedb" \
	--normal-sample-name C57BL6_wes_1 \
	--normal-cov 1 \
	--normal-vaf 0.99 \
	--trna-cov 1 \
	--trna-vaf 0.01 \
	--tdna-cov 1 \
	--tdna-vaf 0.01 \
	--expn-val 1.0 \
	--maximum-transcript-support-level 5 \
	-t 16 \
	/cromwell_root/fc-ecef88af-5f5e-4d27-afe3-5c5f95ab8852/c4a4fbab-f2cd-4708-89a9-5721111fb2fd/mouse_variants2neoantigens/f49dcfd6-5251-4076-b443-449499242601/call-VEP/mutect_strelka_union_vep.vcf \
	GL261-8H3_wes_1 \
	H-2-Db,H-2-Kb \
	NetMHC NetMHCpan PickPocket \
	/cromwell_root
```
Input files: 
As the vcf and all_epitopes files are greater than 100 MB in size, perhaps we can share the files over Google Drive? 
I have attached the full log file to this report. 

**Log Output**
```
Combining Parsed Prediction Files
Completed
Creating aggregated report
Traceback (most recent call last):
  File "/usr/local/bin/pvacseq", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/tools/pvacseq/main.py", line 95, in main
    args[0].func.main(args[1])
  File "/usr/local/lib/python3.7/site-packages/tools/pvacseq/run.py", line 122, in main
    pipeline.execute()
  File "/usr/local/lib/python3.7/site-packages/lib/pipeline.py", line 504, in execute
    PostProcessor(**post_processing_params).execute()
  File "/usr/local/lib/python3.7/site-packages/lib/post_processor.py", line 29, in execute
    self.aggregate_all_epitopes()
  File "/usr/local/lib/python3.7/site-packages/lib/post_processor.py", line 44, in aggregate_all_epitopes
    AggregateAllEpitopes(self.input_file, self.aggregate_report, self.file_type).execute()
  File "/usr/local/lib/python3.7/site-packages/lib/aggregate_all_epitopes.py", line 212, in execute
    best_mut_line = self.get_best_mut_line(df_subset, hla_types, vaf_clonal, 1000)
  File "/usr/local/lib/python3.7/site-packages/lib/aggregate_all_epitopes.py", line 113, in get_best_mut_line
    (wt_aa, mt_aa) = best["Mutation"].split("/")
ValueError: not enough values to unpack (expected 2, got 1)
```
[pVACseq_PredictionsWithoutRNA.log](https://github.com/griffithlab/pVACtools/files/6326673/pVACseq_PredictionsWithoutRNA.log)

**Expected behavior**
The above command is the default command that I use in a larger pipeline to call the pVACseq pipeline component. I have used the same command parameters on other samples and pVACseq is able to create the aggregated report without running into an error.

**Comment by susannasiebert** [comment]
Thank you for reporting this error and I apologize that you are running into problems. This appears to be some sort of edge case with the data you are running, possibly related to this being mouse data. We haven't done much testing on this new report with non-human data. Would you be able to share your input VCF with us for further debugging?

**Comment by nruthen** [comment]
I appreciate you helping me resolve this issue! I just shared one of the input VEP-annotated VCF files that triggered the error with your ssiebert.gc@gmail.com account. Please let me know if you need additional information or files.

**Comment by susannasiebert** [comment]
I was able to determine the reason for your error. A pull requires for the fix is at #662. Please see that PR for a more thorough explanation of the problem. 

This will require a new version of pVACtools to resolve. Once the new version is out (probably by the end of the week) you should be able to rerun pVACseq on your existing output directory and it should pick back up where it failed previously and won't need to recalculate all of the binding predictions.

**Comment by susannasiebert** [comment]
I just released pVACtools version 2.0.2. Please give it a try and let me know if it resolved your problem. If you still have the output directories from the previous run you should just be able to rerun it and it will pick up where it previously failed.

**Comment by nruthen** [comment]
The update in pVACtools version 2.0.2 seems to have resolved the problem. The above sample was able to run through the pVACseq pipeline without error. Thank you for all your help!
